### ISSUE TITLE
[Bug]: NoneType error when using remote Ollama

### ISSUE THREAD

**Issue by phillips-systems** [danielmiessler/Fabric#365]
### What happened?

On a default fabric install on Ubuntu 23.10 I'm receiving the following error using a remote Ollama server:

```
cat /tmp/transcript | fabric -p create_micro_summary --remoteOllamaServer http://<REDACTED>:11434 --model llama3
Error: 'NoneType' object has no attribute 'chat'
'NoneType' object has no attribute 'chat'
```

No error is received when using OpenAI:

```
cat /tmp/transcript| fabric -p create_micro_summary
ONE SENTENCE SUMMARY:
Stefan experiments with a novel 3D printing slicing method that mimics brick laying for stronger layer adhesion.

MAIN POINTS:
- Regular 3D prints have aligned extrusions, reducing layer adhesion.
- Stefan's method involves interlocking perimeters, increasing strength.
- Tests show a 10-14% increase in strength with the new method.

TAKEAWAYS:
- Interlocking pattern improves 3D print strength.
- Novel slicing approach shows significant potential.
- Experimentation and optimization can advance 3D printing technology.

```

### Version check

- [X] Yes I was.

**Comment by strikeoncmputrz** [comment]
I was able to reproduce this. I think the issue here may be that you are not specifying the full name of the Ollama model.  Here are two examples. The first one reproduces the issue, the second includes the full name and does not.

Same issue:
```
$ pst | summarize --remoteOllamaServer http://llama.lan:11434 --model wizardlm2
Error: 'NoneType' object has no attribute 'chat'
'NoneType' object has no attribute 'chat'
```
Successful prompt:

```
$ pst | summarize --remoteOllamaServer http://llama.lan:11434 --model wizardlm2:8x22b-q4_0

The op-ed by Sylvia Benito raises important concerns about.....
```
Also, thanks very much for your example. It wasn't obvious to me how to get a remote Ollama server to work since fabric can't query the remote Ollama API to determine which models are available. With your example I was able to get up and running. Now time to set some env variables!
